prow/cluster/deck_service.yaml

```diff
@@ -15,12 +15,18 @@
 apiVersion: v1
 kind: Service
 metadata:
+  labels:
+    app: prow
```

Comment:
In this repo, there is no  `app` and `component` -- let's use the convention just with `app`. So this would need `app: deck`